seattleblmcalendar.rss

```diff
@@ -0,0 +1,515 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I don't think we want to include these two files

Reply:
Definitely not, removed em!